epics-team-builder/src/sass/widgets/setup-page.scss

```diff
@@ -91,6 +91,27 @@
     grid-row: 4/18;
   }
 
+  .assign-students-button{
+    box-sizing : border-box;
+    width: 130px;
+    text-align: center;
+    color: white;
+    background-color: #ec3d26;
+    padding: 2px 8px;
+    border-radius: 8px;
+    font-weight: bold;
+    font-size: 16px;
+    border: solid white 3px;
+    box-shadow: 2px 4px 4px 0 rgba(0, 0, 0, 0.5);
+    vertical-align: middle;
+    cursor: pointer;
+
+    &:focus {
+      box-shadow: 0 0 3px rgba(0, 0, 0, 0.8);
+      transition: box-shadow 0.3s ease-in-out;
+    }
+  }
+
```

Comment:
I think we can just turn this into a common css class since we will be using this style for buttons everywhere. Something similar to this https://blog.teamtreehouse.com/create-a-themable-button-set-with-sass can be done. This allows us to just provide a color as a class in the button and have the same style everywhere.